tensorflow/core/platform/default/logging.cc

```diff
@@ -81,7 +82,38 @@ void LogMessage::GenerateLogMessage() {
 }
 #endif
 
-LogMessage::~LogMessage() { GenerateLogMessage(); }
+
+namespace {
+
+int64 MinLogLevel() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_LOG_LEVEL");
+  if (tf_env_var_val == nullptr) {
+    return 0;
+  }
+
+  // Ideally we would use env_var / safe_strto64, but it is
+  // hard to use here without pulling in a lot of dependencies,
+  // so we do a poor-man's parsing.
+  StringPiece min_log_level_sp(tf_env_var_val);
+  string min_log_level_str(min_log_level_sp.data(), min_log_level_sp.size());
+  if (min_log_level_str == "1") {
```

Comment:
How about also their actual names? For example, case insensitive, "info", "warning", "error", etc.